### DISCUSSION TITLE
Issue TabBar with Shell NET MAUI

### DISCUSSION THREAD

**Discussion by Javierflo95** [dotnet/maui#6958] [Q&A] [answered]
Hello all,
I hope you are well,
I'm trying to do a TabBar with Shell but i'm getting an issue and I've couldn't solve it, someone can help me pls thanks.

@jamesmontemagno 
@jfversluis 

Here my code 
![image](https://user-images.githubusercontent.com/11860518/167453035-f31bd050-1e50-4171-b8bb-e33ae2080839.png)
--
![image](https://user-images.githubusercontent.com/11860518/167453147-a3865e47-2b70-437e-9637-5ad9a020aa2b.png)
---
![image](https://user-images.githubusercontent.com/11860518/167453173-292ccd61-8f54-45c3-bdcf-24921f67b69e.png)

**Comment by Javierflo95** [comment, marked as answer]
I figured out, including Routing.RegisterRoute(nameof(HomeShellPage), typeof(HomeShellPage)); in ctor where the shell page is called and doind this my HomeShellPage is showing well its content

**Reply by jfversluis** [reply]
Great! Glad you figured it out
